packages/ddp-client/stream_client_common.js

```diff
@@ -22,6 +22,10 @@ var translateUrl =  function(url, newSchemeBase, subPath) {
     newSchemeBase = "http";
   }
 
+  if (subPath !== "sockjs" && startsWith(url, "/")) {
```

Comment:
Really interesting that `toWebsocketUrl` didn't work as-is - maybe the way server-side faye-websocket client works is different from the browser native Websocket.